blog/settings.py

```diff
@@ -3,11 +3,10 @@
 import environ
 
 env = environ.Env(DEBUG=(bool, False),)
-root = environ.Path(__file__) - 3
-environ.Env.read_env()
-# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
-BASE_DIR = root()
 
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = environ.Path(__file__) - 2
+env.read_env(str(BASE_DIR.path('.env')))
```

Comment:
Have you tried doing it [this way](https://github.com/open-craft-guild/crm-smsfly/blob/master/SMSFlyCRM/settings.py#L17)?